docs/pt/docs/learn/index.md

```diff
@@ -0,0 +1,5 @@
+# Aprender
+
+Nesta parte da documentação encontramos as seções introdutórias e os tutoriais para aprendermos **FastAPI**.
+
+Nós poderíamos considerar isto um **livro**, **curso**, a maneira **oficial** e recomendada de aprender a FastAPI. 😎
```

Comment:
**FastAPI** is an abstraction layer over the Python programming language to facilitate the development of Web APIs or Web applications.

The *abstraction* is translated to **abstração**, so from there comes the feminine defined article. But if **framework** would be the preferred word here, we'll fall in the same solution because **framework** is translated for many as **estrutura**, but **abstração** is more accurate since the framework itself offer something more than a simple structure.

@codespearhead.

Reply:
In Brazilian Portuguese we don't have a word for `framework` in the context of Computer Science. If we need to describe it generically for someone that's not in the field we call it `a ferramenta`, but other than that it's always a masculine now, as seen in [here](https://learn.microsoft.com/pt-br/dotnet/framework/get-started/) and [here](https://www.ibm.com/br-pt/topics/java-spring-boot).